src/app/aircraft/edit/index.njk

```diff
@@ -16,81 +16,56 @@
   <div class="govuk-breadcrumbs">
     <ol class="govuk-breadcrumbs__list">
       <li class="govuk-breadcrumbs__list-item">
-        <a class="govuk-breadcrumbs__link" href="/aircraft">Aircraft</a>
-      </li>
-      <li class="govuk-breadcrumbs__list-item">
-        Edit aircraft
+        <a class="govuk-breadcrumbs__link" href="/aircraft">{{__('title_aircraft')}}</a>
       </li>
+      <li class="govuk-breadcrumbs__list-item">{{__('title_edit_aircraft')}}</li>
     </ol>
   </div>
 
 {% endblock %}
 
 {% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if errors %}
+        {% include "common/templates/includes/errors.njk" %}
+      {% endif %}
 
-  {% import "common/templates/includes/macros.njk" as m %}
-
-<div class="govuk-grid-row">
-	<div class="govuk-grid-column-two-thirds">
-
-	{% if errors %}
-    {% include "common/templates/includes/errors.njk" %}
-  {% endif %}
-
-      <form action="/aircraft/edit" autocomplete="off" class="form-group"
-      method="post" role="form" >
+      <form action="/aircraft/edit" autocomplete="off" class="form-group"method="post" role="form" >
 
-      <h1 class="govuk-heading-xl">
-        Edit aircraft
-      </h1>
+      <h1 class="govuk-heading-xl">{{__('title_edit_aircraft')}}</h1>
 
-      <p class="govuk-body-l">Please enter the following information. This information can be re-used when submitting a GAR.</p>
+      <p class="govuk-body-l">{{__('edit_aircraft_caption')}}</p>
 
-      <div class="govuk-form-group {{ m.form_group_class('craftReg', errors) }}">
-          <label class="govuk-label" for="craftReg">Aircraft registration</label>
-          {{ m.error_message('craftReg', errors, __) }}
-          <span class="govuk-hint">
-            For example G-ABCD
-          </span>
-          <span class="field-validation-valid error-message" data-valmsg-for="craftReg"
```

Comment:
This span block is some hangover from either JQuery unintrusive validation or ASP.net, both not used in this app.